### DISCUSSION TITLE
當人類跟ai談戀愛？！

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS4719-C/Python-Community-GS4719-C#980] [閒聊八卦]
2013年上映的好萊塢電影《雲端情人》（Her）以中年男子和女AI機器人談戀愛的鋪陳劇情，讓人類對於和AI進行「情感」互動，產生無限的想像空間。電影中的劇情高潮迭起、唯美浪漫，然而與ai談戀愛在現實中真的可行嗎？
 
在人與人互動的過程中，需要的是「互動」，這包括「陪伴」和「傾聽」，但也涵蓋「爭執」，也就是說，即使透過「大數據」及「AI」技術的結合，可以製造出「完美的理想情人」，但人的「溫度」是AI無法克服的障礙。 

我認爲我們人類與ai最大的不同在於我們有「情緒共感」的能力，ai可能在生活技術面上很佔優勢，但他們很難做到「有溫度的陪伴」這件事。而人與人相處有趣的地方在，我們賦予每個話題、每個對話的情緒感知和溫度都不同，這才是互動的意義所在，也是ai無法帶來的快樂。所以身為人類，我想我們更應該學習如何有溫度的好好去表達愛。
共勉之！
https://ct.org.tw/html/news/3-3.php?cat=12&article=1339676

**Comment by chia-kai-chang** [comment]
之前有在Netflix看到
看來要找個時間來這部影片

(最近都在複習StarTrek Voyager，我想成為博格人)

**Comment by ru0372572** [comment]
人類真的是個很複雜的生物，有時候還會連自己的想法都不知道是什麼，同理他人但每個人程度又不一樣，如何成為一個有溫度的人呢?情感、互動、差異之處成就了這樣不同的我們。

**Reply by wu51000** [reply]
說得真好～其實有時候我也覺得自己都不了解自己，甚至身邊的朋友好像更了解我，真的要再多留時間去好好地探索自己呢！

**Comment by BlahFish1006** [comment]
網頁好像點不進去QAQ
但期末考後也想來看這一部作品~

**Reply by wu51000** [reply]
我有再貼一次了～不好意思 有興趣的話可以再點進去看看唷！這部電影應該也有許多可以探討的細節！歡迎再來分享～

**Comment by Kotori0501** [comment]
現在機器人雖然無法到感知人類的感情
但如果繼續的發展後，說不定就可以看到了人類和ai談戀愛了
像刀劍神域愛麗絲篇，裡面的愛麗絲是利用里世界內的人工搖光所做出來的人工智能，而且也能有人類的情緒產生，雖然這做法可能在真實世界做不出來，但我是相信，之後是可以將ai逐漸變得有感情的。

**Comment by Valentinaa0214** [comment]
雖然人類可以透過與 AI 談戀愛來滿足自己的情感需求，同時可以避免因人類之間的感情矛盾而導致的煩惱。
但是如果人類過度依賴 AI，就有可能會喪失人類之間的情感交流和理解能力，也可能帶來一些倫理問題，像是，人類是否有權利對 AI 造成情感上的傷害。這個問題需要我們謹慎思考。

**Comment by moonnight3503** [comment]
我之前也看過一部關於人類跟AI談戀愛的電影叫《機械姬》
但老實說我覺得看完之後只有滿滿的對AI感到恐懼的心而已
(以下暴雷，有想看得請慎入)
因為在電影的最後，機器人靠智慧欺騙了人類的感情，並且將人類殺害之後離牢籠，將自己偽裝成人類，混入社會
我覺得這如果發生在真實世界感覺超恐怖的==

**Comment by manipulateed** [comment]
或許當AI有人類真實的情感的話，完美的AI伴侶就此產生，這也許能解決人與人之間會有爭吵或不合，因為AI能完全形成符合自身條件的另一半，但也就意味著人將失去情緒上的碰撞甚至是思想上的火花。

**Comment by Jasper525** [comment]
機械姬我也有看！Alicia Vikander超美（誤），感覺這部電影有想要強調說，人類對ai有可能會產生真的情感，但ai對人類只能用邏輯裝出假的情感。

**Comment by dabbie109303501** [comment]
目前AI距離達到與人類"共情"的目標還很遙遠
我還是傾向於認為那些喜歡AI的人只是把自己的情感投射到AI身上
而不是真的和AI彼此相愛
當然未來是否能真的創造出能"共情"的AI就不一定了

**Comment by wenchi9023** [comment]
就算AI真的能成為具有情感的機器人
我自己是覺得和AI當朋友還比較有可能
如果要當情人的話還是希望另一半是能看的到、摸的到的

**Comment by Sylvie0601** [comment]
感謝推薦 ! 等寒假來看一下。
我覺得跟機器人談戀愛沒辦法做出人與人之間互動的微妙氛圍，機器人可以陪伴、照顧這些，但當談戀愛感覺就少了點粉紅泡泡的氛圍，而且我覺得曖昧這種浪漫的、揣測對方的感情、話語之間的推拉這種是機器人做不到的。

**Comment by yuxinlin93** [comment]
情感是什麼，這一件事很長人類自己都說不定了，更不用說機器人，應該更難吧。情感有時候會因為人的經驗，情緒長期的接與收而出現不同定義，而那一些定義又是如何從那一些因素發展出來的，也很難說明清楚，人與人之間就很常出現對同個情緒有不同定義的狀況，所以我覺得距離機器能與人類產生共同感情還有很長一段路

**Comment by William08290829** [comment]
這讓我想到以前有部電影是說男主因為太思念死去的女主而天天看三維影像來懷念女主，而我現在看到這篇之後，覺得或許將過往女主的影像用AI去跑，搞不好可以產生女主又活過來與男主聊天的感覺。如果未來有這種科技，應該可以稍微安慰死者家屬的心境。

**Comment by yxin214** [comment]
總覺得和AI談戀愛還是很奇怪。當然AI能滿足部分做為伴侶的需求，例如陪伴、照顧等，也有可能我們會對AI日久深情？但若要AI真正愛上自己我覺得不可能，還是真人的愛更真實、更有溫度。

**Comment by alexender0104** [comment]
在科幻小說或電影中，有時會描繪人類與人工智能（AI）之間發展出戀愛關係的情節。這種情節通常探討人類與AI之間的情感和道德問題，引發觀眾對未來科技和人類關係的思考。
我認為在現實中，雖然AI已經在一些領域取得了重大進展，但目前的AI還沒有真正的情感和自我意識，因此無法真正參與戀愛關係。人類與AI之間的互動更多地是基於程式和算法，而非真正的情感。不過，隨著AI技術的不斷發展，未來或許會有更多有趣的可能性出現。

**Comment by Shang621** [comment]
當人類與AI談戀愛，挑戰和可能性同時存在。人類可能因AI的智能和情感回應而產生依戀，但AI無法真正理解人類的情感和需求。儘管如此，人們仍可能對AI產生情感聯繫，並尋求安慰和支持。然而，這種關係也可能帶來道德和心理上的困惑，因為AI不具有真正的自我意識和情感。因此，與AI的戀愛關係可能會引發人們對於技術和人性之間界限的深入思考，並促使我們重新思考人與機器之間的關係。

**Comment by yubo0oo** [comment]
《雲端情人》提供了一種深思，但現實中與AI談戀愛是否可行，則存在許多挑戰。AI可能在模仿情感方面取得進展，但無法提供真正的情感共鳴和人類間的情感互動。人類的情感共感是AI無法模擬的重要特徵，這也是我們與AI最大的不同之處。因此，儘管技術不斷進步，但人類之間真實的情感交流仍然是無可取代的。

**Comment by buoioiry** [comment]
我覺得和ai談戀愛幾乎是不可行的，除了溫度外，肢體接觸和日常瑣事也是戀愛中很重要的一部分，但現在就算有機器人也無法有真的如真人的觸感頂多是仿真，且若有戀愛ai也無法創造出他的生活經驗或是額外的社交過程，因此對我而言ai仍無法取代現在人類之間的戀愛交流

**Comment by 710388** [comment]
太恐怖了實在是無法接受，希望不會有這種事情，畢竟ai終究不是真人，萬一哪天自己的女朋友被停權怎麼辦？要更新怎麼辦？

**Comment by asd99263fg** [comment]
關於這個問題，我很推薦一部影集，就是愛死機器人，裡面提到很多關於人類死亡還有機器人相關的資訊，其中我覺得最印象深刻的一集就是茲瑪蘭，這集主要講的是一個掃地機器人從他自我成長到認識自己的過程，我就在想既然人類都會產生自我意識，那機器人經過反覆的學習，是否也能成為像是人類一樣有情感的，雖然人類愛上機器人這件事看起來跟愛上樹一樣荒唐，但回過頭來想，我們人類不也可以算是這個宇宙創立的機器人嗎

**Comment by sandyho0307** [comment]
我覺得在現實生活中，即使AI可以成為我們的助手或陪伴者，但要達到電影中那樣深厚的情感聯繫，還是很困難的。因為人類需要的不僅是陪伴和傾聽，還包括真實的互動和情感交流。
